src/Backdrop/Subheader.js

```diff
@@ -14,7 +21,11 @@ const Subheader = ({ disabled, onPress, icon, title, titleStyle }) => {
       accessibilityRole="button"
     >
       <View style={styles.headerContainer}>
-        <Text style={[styles.title, titleStyle]}>{title}</Text>
+        {TitleComponent ? (
```

Comment:
@enieber thanks for opening a PR 😄. How about instead of adding this new prop we use the title for that? The title would accept a React component or a string.

```suggestion
        React.isValidElement(title) ? {title} : <Text style={[styles.title, titleStyle]}>{title}</Text>
```

I think this might work, can you double-check?

Reply:
but has one problem to this mode. Is necessary add title acessibility text, if component title can be component so is necessary add new prop to acessibility text because accessibilityLabel receive title prop:

`accessibilityLabel={title}`